src/AltinnCore/Common/Configuration/ServiceRepositorySettings.cs

```diff
@@ -17,6 +17,11 @@ public class ServiceRepositorySettings
         /// </summary>
         public const string RESOURCE_FOLDER_NAME = "Resources/";
 
+        /// <summary>
+        /// Constant for the location of resource files
```

Comment:
the description for this value is the same as for RESOURCE_FOLDER_NAME, it should be changed to avoid confusion

Reply:
Fixed

Reply:
@nkylstad I cant see that this is acctually fixed

Reply:
@hannekot fixed in another line 😺 updated this line now